lib/python/satos_payload_sdk/antaris_api_gpio.py

```diff
@@ -76,3 +76,26 @@ def api_pa_pc_write_gpio(pin, value):
     time.sleep(g_SLEEP_TIME_IN_SEC)
     op = (Device.port >> pin) & g_MASK_BIT_0
```

Comment:
This file provides api to make gpio pins high/low.
In Janus-1, one of payload wanted "high-low-high" pulse. So, this file provides api for that.

Reply:
Understood. I am just placing the example usage in the file for now until we have a chance to place it more appropriately.